### PR TITLE
add support for empty wifi passwords

### PR DESCRIPTION
**Pull request by abretro**
I've tested this with a blank password profile and a regular password profile, in addition to connecting to a passwordless DHCP network after scanning.

### CODE REVIEW

**Comment by xonglebongle** [comment]
You will have to rebase from HEAD as your PR currently removes quite a number of recent changes from commits.

File: muxnetwork/main.c [hunk 1]
```diff
@@ -283,7 +285,7 @@ void init_navigation_groups() {
     apply_theme_list_panel(&theme, &device, ui_pnlDNS);
     apply_theme_list_panel(&theme, &device, ui_pnlConnect);
 
-    apply_theme_list_item(&theme, ui_lblEnable, TS("Network Active"), false, true);
+    apply_theme_list_item(&theme, ui_lblEnable, TG("Network Active"), false, true);
```
Comment (xonglebongle) [inline muxnetwork/main.c:288]: This needs to be put back to TS

File: muxnetprofile/main.c [hunk 2]
```diff
@@ -464,8 +465,6 @@ void ui_refresh_task() {
         if (lv_group_get_obj_count(ui_group) > 0) adjust_wallpaper_element(ui_group, 0);
         adjust_panel_priority(ui_mux_panels, sizeof(ui_mux_panels) / sizeof(ui_mux_panels[0]));
 
-        lv_obj_move_foreground(overlay_image);
```
Comment (xonglebongle) [inline muxnetprofile/main.c:467]: Please place this and the next line back

Comment (abretro) [reply muxnetprofile/main.c]: all of the cleanup requests are complete. Testing note: you need the change I posted a PR for in muos/internals along with this PR's changes to muxnetwork and muxnetprofile's main.c files for empty wifi passwords to work.

**Pull request merged**